app/routes/wow.quantity-manipulation.tsx

```
@@ -259,130 +279,150 @@ const Index = () => {
+              }
+            />
+            <ItemQualitySelect
+              defaultValue={loaderData.itemQuality}
+              onChange={(value) => handleFormChange('itemQuality', value)}
+            />
+            <ExpansionSelect
+              defaultValue={loaderData.expansionNumber}
+              onChange={(value) => handleFormChange('expansionNumber', value)}
+            />
+            <ItemClassSelect
+              itemClass={parseInt(loaderData.itemClass)}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)